File: Scripts/volatile_and_force_compiler_task/inheritance-multiple/python/inheritance-multiple.py
class Camera:
    def shootPicture(self):
        print "Say cheese\n"

class MobilePhone:
    def call(self):
        print "Pick a call\n"

class CameraPhone(Camera, MobilePhone):
    def activity(self):
        self.shootPicture()
        self.call()

for i in range(1000000):
	obj = CameraPhone()
	obj.activity()

File: Scripts/volatile_and_force_compiler_task/inheritance-multiple/perl/inheritance-multiple.pl
use MooseX::Declare;

class Camera {
    # methods ...
    sub shootPicture {
        print "Say cheese\n";
    }
}
class MobilePhone {
    # methods ...
    sub call {
        print "Pick a phone call\n"
    }
}
class CameraPhone extends(Camera, MobilePhone) {
    # methods ...
}

for( $a=0; $a < 1000000; $a = $a + 1) {
    my $obj = CameraPhone->new();
    $obj->shootPicture();
    $obj->call();
}
